Dockerfile

```diff
@@ -7,7 +7,7 @@ FROM centos:8 as tuned
 WORKDIR /root
 COPY assets /root/assets
 RUN INSTALL_PKGS=" \
-      gcc git rpm-build make desktop-file-utils patch dnf-plugins-core \
+      gcc git rpm-build make desktop-file-utils patch dnf-plugins-core kernel-tools-libs-4.18.0-305.3.1.el8_4.x86_64 kernel-tools-4.18.0-305.3.1.el8_4.x86_64 python3-perf-4.18.0-305.3.1.el8_4.x86_64 \
```

Comment:
`FROM centos:8 as tuned` tuned here is just a "build" container for building `TuneD` rpms.  They're later used in the operator/operand container instantiated from the image base `registry.ci.openshift.org/ocp/4.9:base`.  So I fail to see how "pinning" `kernel-tools-libs` and `kernel-tools` can help here.  Also the upstream `Dockerfile` is not used by the CI at all.